src/cli.ts

```diff
@@ -141,7 +143,7 @@ function runWithOptions(options) {
       if (options['config-file'] && options['config-field']) {
         var field = options['config-field'];
         try {
-          var contents = fs.readFileSync(options['config-file']);
+          var contents:any = fs.readFileSync(options['config-file']);
```

Comment:
Favor let over var wherever possible.

Create a new variable name instead of avoiding type checks for contents.